README.md

```diff
@@ -0,0 +1,25 @@
+# EntityUtils
+
+_TODO write the README_
+
+## Skip validations for production
+
+You can skip validations for production by adding a initializer
+
+```ruby
+# config/initializer/entity_utils.rb
+
+EntityUtils.configure!(
+  validate: false
+)
+```
+
+Setting `validate: false` will now skip all the validation by default. You can override this setting locally by passing the `validate` option to the `define_builder` method:
+
+```ruby
+person = EntityUtils.define_builder(
+  [:name, :string, :mandatory],
+  [:age, :fixnum, :mandatory],
+  validate: true
+)
+```
```

Comment:
The method signature is getting quite complex (as is evident from the config parsing code at implementation side too). If you want to draw inspiration from Plumatic/schema at Clojure side then you could instead have a different function to create an always validating flavour. Schema also has something that would be in Ruby similar to:

```ruby
EntityUtils.with_validations {
  run_some_code_using_builders()
}
```

That is, turn the validation on for a block.